codes

```diff
@@ -0,0 +1,393 @@
+  height: 50px;
+}
+
+/* h1 text */
+.title1 {
+  font-family: 'inter', sans-serif;
+  font-size: 15px;
+  font-weight: 100;
+  font-style: normal;
+  padding-top: 8px;
+}
+
+/* icon */
+i {
+  border: none;
+  background: none;
+  display: flex;
+  justify-content: flex-end;
+  cursor: pointer;
+}
+
+/* task list base */
+#inputList {
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  height: 50px;
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+.inputTask {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-left: none;
+  border-bottom: none;
+  border-right: none;
+  border-top: 1px solid gray;
+  height: 50px;
+  padding: 0 10px;
+}
+
+/* refresh icon */
+.inputTask i {
+  rotate: 90deg;
+  cursor: pointer;
+}
+
+/* input text  */
+#addList {
+  width: 100%;
+  height: 45px;
+  border: none;
+}
+
+.submiticon {
+  padding: 2px;
+}
+
+.remove-edit {
+  display: flex;
+  justify-items: flex-end;
+  margin-left: auto;
+  padding-right: 23px;
+  text-decoration: none;
+}
+
+/* Clear task button */
+#btnClear {
+  height: 50px;
+  color: rgb(147, 147, 147);
+  border-left: none;
+  border-bottom: none;
+  border-right: none;
+  border-top: 1px solid gray;
+  font-size: 18px;
+  font-family: 'inter', sans-serif;
+  cursor: pointer;
+}
+
+/* task list base */
+.list {
+  background: #fff;
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  height: 50px;
+  border-top: 1px solid gray;
+  padding-left: 10px;
+  list-style: none;
+}
+
+/* text */
+.tasks label {
+  font-family: 'inter', sans-serif;
+  padding-left: 10px;
+  width: 100%;
+}
+
+/* text */
+.lists li:nth-child(3) {
+  display: flex;
+  justify-items: flex-end;
+  margin-left: auto;
+  padding-right: 15px;
+  text-decoration: none;
+}
+
+/* text checkbox */
+input[type=checkbox]:checked + label {
+  text-decoration: line-through;
+}
+
+/* text */
+input {
+  font-family: 'inter', sans-serif;
+  font-size: 15px;
+  font-style: italic;
+  font-weight: 100;
+}
+
+/* text */
+input:focus {
+  outline: none;
+}
+
+/* text */
+.text {
+  height: 45px;
+  width: 95%;
+  padding-left: 5px;
+  border: none;
+  font-family: 'inter', sans-serif;
+  font-size: 15px;
+}
+
+.textcompleted {
+  text-decoration: line-through;
+  border: none;
+  font-size: 17px;
+  height: 45px;
+  width: 95%;
+  padding-left: 5px;
+  color: #9b9b9b;
+}
+
+/* edit button */
+.edit_list_btn {
+  cursor: pointer;
+  border: none;
+  background: none;
+}
+
+/* remove button */
+.remove_btn {
+  cursor: pointer;
+  display: none;
+  border: none;
+  background: none;
+}
+
+.checkbox {
+  cursor: pointer;
+}
+
+.to-do {
+  background: #fff;
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  padding-left: 10px;
+  list-style: none;
+  border-bottom: 0.01px solid #d3d3d3;
+}
+
+//js
+import Data from './data.js';
+
+// get inputs from local storage
+
+export default class display {
```

Comment:
### Good job Implementing project requirements.
Just there are a few changes to be made

1. Create separate files for HTML, CSS, and JavaScript code, they should be in separate files and referenced accordingly